pylivetrader/backend/alpaca.py

```diff
@@ -495,48 +499,49 @@ def get_bars(self, assets, data_frequency, bar_count=500):
             symbols = [assets.symbol]
         else:
             symbols = [asset.symbol for asset in assets]
+        # the timezone of end_dt if UTC, when data_frequency=daily, end_dt is midnight in UTC time
+        if not end_dt:
```

Comment:
why do we need this? end_dt is always now. do you experience issues with this?

following comments are regardless of this question and only touch on code quality